### PR TITLE
(maint) Use CODEOWNERS file

### PR DESCRIPTION
**Pull request by justinstoller**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# This will cause the puppetserver-maintainers group to be assigned
+# review of any opened PRs against the branches containing this file.
```
Comment (justinstoller) [inline CODEOWNERS]: that's a mouthful

**Comment by justinstoller** [comment]
(I manually assigned the maintainers group this time)

**Comment by joshcooper** [comment]
Want to delete https://github.com/puppetlabs/puppetserver/blob/master/MAINTAINERS?

**Comment by justinstoller** [comment]
Kinda just want to rename it to ALUMNI  (;

**Pull request merged**